### ISSUE TITLE
Release 4.6.2 isn't bitcode compatible

### ISSUE THREAD

**Issue by jilouc** [teads/TeadsSDK-adapter-admob-ios#3]
Since upgrading to TeadsSDK 4.6.2 / TeadsSDK adapter for Admob 4.6.2, we have trouble uploading to the AppStore. It is apparently some Bitcode issue. Here's what we received from Apple:

> ITMS-90562: Invalid Bundle - There's an issue with the app's bitcode compilation. Rebuild the app using the current public (GM) version of Xcode and upload it again.
> 
> ITMS-90562: Invalid Bundle - The app submission can not be successfully recompiled from bitcode due to missing symbols during linking. You can try to reproduce and diagnose such issues locally by following the instructions from: https://developer.apple.com/library/archive/technotes/tn2432/_index.html

Following the troubleshooting steps in Apple documentation we get this error:
```
ipatool failed with an exception: 
#<CmdSpec::NonZeroExitException: $ /Applications/Xcode.app/Contents/Developer/usr/bin/python3 
/Applications/Xcode.app/Contents/Developer/usr/bin/bitcode-build-tool -v -t 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin 
-L /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsSDK.framework 
-L /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks 
--sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.4.sdk 
-o /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter 
--generate-dsym /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-out/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter.dSYM 
--strip-swift-symbols /var/folders/24/lv2nv0v95l748x6x02n812gw0000gn/T/ipatool20200428-95813-i91xsg/thinned-in/arm64/Payload/XXXXX.app/Frameworks/TeadsAdMobAdapter.framework/TeadsAdMobAdapter
Status: pid 4372 exit 1
```

Checking the TeadsAdMobAdapter.framework for Bitcode as suggested here https://forums.developer.apple.com/message/7038

```
otool -l TeadsAdMobAdapter.framework/TeadsAdMobAdapter | grep __LLVM
```
returns no result

(it does for the Teads main SDK)

Disabling bitcode is not an option as we have a watchOS app.

**Comment by jilouc** [comment]
Looks like the Instabug SDK had the same issue a while ago
https://github.com/Instabug/Instabug-iOS/issues/322

**Comment by elneruda** [comment]
Hello @jilouc thank you for submitting this issue, I can confirm behaviour : bitcode on Admob adapter is not available on this release, we are investigating on it and will release a fix soon

**Comment by elneruda** [comment]
Hello @jilouc we have just released a new version of [TeadsAdMobAdapter 4.6.3](https://github.com/teads/TeadsSDK-adapter-admob-ios/releases/tag/v4.6.3) which supports bitcode, let me know if everything works fine

**Comment by jilouc** [comment]
I confirm that 4.6.3 fixes the bitcode issue, thanks @elneruda 👍
